core/src/main/java/com/linecorp/armeria/server/AnnotatedValueResolver.java

```diff
@@ -245,11 +273,16 @@
      * @param pathParams       a set of path variables.
      * @param objectResolvers  a list of {@link RequestObjectResolver} to be evaluated for the objects which
      *                         are annotated with {@link RequestObject} annotation.
+     * @param noAnnotationAsRequestObject a flag which indicates how to handle an element which is not
+     *                                    annotated. If it is {@code true}, it would be handled the same as
+     *                                    when a {@link RequestObject} is specified. Otherwise, the element
+     *                                    would be ignored.
```

Comment:
Not sure this is the best description, but:

```
@param implicitRequestObjectAnnotation {@code true} if an element is always treated like it is annotated
                                       with {@link RequestObject} so that conversion is performed.
                                       {@code false} if an element has to be annotated with
                                       {@link RequestObject} explicitly to get converted.
```